cmd/agent/app/builder.go

```diff
@@ -128,16 +129,18 @@ func (b *Builder) getReporter(primaryProxy CollectorProxy) reporter.Reporter {
 	return reporter.NewMultiReporter(rep...)
 }
 
-func (b *Builder) publishOpts(mFactory metrics.Factory) {
-	internalFactory := mFactory.Namespace(metrics.NSOptions{Name: "internal"})
+func (b *Builder) publishOpts() {
 	for _, p := range b.Processors {
 		prefix := fmt.Sprintf(processorPrefixFmt, p.Model, p.Protocol)
-		internalFactory.Gauge(metrics.Options{Name: prefix + suffixServerMaxPacketSize}).
-			Update(int64(p.Server.MaxPacketSize))
-		internalFactory.Gauge(metrics.Options{Name: prefix + suffixServerQueueSize}).
-			Update(int64(p.Server.QueueSize))
-		internalFactory.Gauge(metrics.Options{Name: prefix + suffixWorkers}).
-			Update(int64(p.Workers))
+		safeexpvar.SetExpvarInt(prefix+suffixServerMaxPacketSize, int64(p.Server.MaxPacketSize))
```

Comment:
```suggestion
		safeexpvar.SetInt(prefix+suffixServerMaxPacketSize, int64(p.Server.MaxPacketSize))
```
it is already explicit from package name that this is expvar